docs/clientsidequeries.md

```diff
@@ -0,0 +1,85 @@
+---
+id: clientqueries
+title: Client-side Queries
+---
+
+Additionaly graphback also allows users to generate client-side queries based on the data model.
```

Comment:
Missing info that users still need to follow up with any available clients like Apollo and use something like create-react-app to kickstart experience.